airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1979,258 +1978,6 @@ def request_params(
         return params
 
 
-class PropertyHistory(ClientSideIncrementalStream):
-    """Contacts Endpoint, API v1
-    Is used to get all Contacts and the history of their respective
-    Properties. Whenever a property is changed it is added here.
-    Docs: https://legacydocs.hubspot.com/docs/methods/contacts/get_contacts
-    """
-
-    updated_at_field = "timestamp"
-    created_at_field = "timestamp"
-    denormalize_records = True
```

Comment:
Want to note that due to `denormalize_records = True`, this used to trigger a flow where we did not merge property records together by their PK. However, we do have a valid PK to merge on so it made sense to update the behavior to merge if records do match.

In the end I don't think this makes a difference functionally because we are already emitting each property individually from one another in low-code chunking